wsmaster/codenvy-hosted-api-permission/src/main/java/com/codenvy/api/permission/server/PermissionsManager.java

```diff
@@ -73,6 +77,7 @@ public PermissionsManager(EventService eventService,
         }
         this.domains = ImmutableList.copyOf(domains);
         this.domainToDao = ImmutableMap.copyOf(domainToDao);
+        this.updateLocks = new StripedLocks(16);
```

Comment:
Does it really have to deal with distributed locks?
It looks like a single lock instance should be fine.